power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/vnf_pgm_converter.h

```diff
@@ -28,8 +28,7 @@ PGM_IO_API PGM_IO_VnfConverter* PGM_VNF_create_converter(PGM_IO_Handle* handle,
  * @param dataset A pointer to the const dataset supplied by the user.
  * @return The pointer to the const dataset instance supplied by the user which has been filled in.
  */
-PGM_IO_API PGM_IO_ConstDataset const* PGM_VNF_get_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr,
-                                                             PGM_IO_ConstDataset const* dataset);
+PGM_IO_API char const* PGM_VNF_get_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr);
```

Comment:
maybe out of scope of this PR but please settle on EITHER `PGM_VNF` OR `PGM_IO` prefix. Mixing is not good. My preference would be `PGM_IO`, because the VNF converter is just one of the potentially more IO Native things